src/newSiteAbTest.ts

```diff
@@ -0,0 +1,36 @@
+type CookieConfig = {
+  name: string
+  maxAge: number
+}
+
+export type AbTestConfig = {
+  optimizeExperimentId: string
+  experimentQueryParam: string
+  cookies: {
+    variant: CookieConfig
+    eligible: CookieConfig
+  }
+  redirects: Record<string, string>
+}
+
+export const newSiteAbTest: AbTestConfig = {
+  optimizeExperimentId: 'wmRyD1ofQYSQ5LL5BEH9nw',
+  experimentQueryParam: 'experimentVariantId',
+  cookies: {
+    variant: {
+      name: 'HEDVIG_EXP_NEW_SITE',
+      maxAge: 7 * 24 * 3600 * 1000,
+    },
+    eligible: {
+      name: 'HEDVIG_EXP_NEW_SITE_ELIGIBLE',
+      maxAge: -1,
+    },
+  },
+  redirects: {
+    '/se-en': '/en-se',
+    '/se-en/insurances/home-insurance/rental': '/en-se/products/home',
+    '/se-en/insurances/car-insurance': '/en-se/products/car',
```

Comment:
nit: these will be updated to `en-se/insurances/[insurance]` for new Hedvig.com but let's change that then